admin/app/components/solidus_admin/option_types/index/component.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="px-4">
```

Comment:
Any reason for not using page helpers here?

Reply:
😅 Missed that, will update soon. Thanks!